packages/boot/test/bootstrapTests/test-liquidation-1.ts

```diff
@@ -389,35 +391,7 @@ test.serial(
 );
 
 test.serial('add STARS collateral', async t => {
-  const { controller, buildProposal } = t.context;
-
-  t.log('building proposal');
-  const proposal = await buildProposal({
-    package: 'builders',
-    packageScriptName: 'build:add-STARS-proposal',
-  });
-
-  for await (const bundle of proposal.bundles) {
-    await controller.validateAndInstallBundle(bundle);
-  }
-  t.log('installed', proposal.bundles.length, 'bundles');
-
-  t.log('launching proposal');
-  const bridgeMessage = {
-    type: 'CORE_EVAL',
-    evals: proposal.evals,
-  };
-  t.log({ bridgeMessage });
-
-  const { EV } = t.context.runUtils;
-  const coreEvalBridgeHandler: ERef<BridgeHandler> = await EV.vat(
-    'bootstrap',
-  ).consumeItem('coreEvalBridgeHandler');
-  await EV(coreEvalBridgeHandler).fromBridge(bridgeMessage);
-
-  t.context.refreshAgoricNamesRemotes();
-
-  t.log('add-STARS proposal executed');
+  await addSTARsCollateral(t);
```

Comment:
given how simple this test is now, I suggest that `checkFlow1` be responsible for adding the collateral, as necessary. It can be aware that ATOM comes from bootstrap. E.g.
```
if (collateralBrandKey !== 'ATOM') {
  addNewCollateral(t, collateralBrandKey);
}
```
Alternately, until there is a general function,
```
if (collateralBrandKey === 'STARS') {
  addSTARsCollateral(t);
}
```

but best yet would be unconditionally,
```
  await ensureVaultCollateral({ collateralBrandKey, managerIndex });
```
That would look at the manager index in vstorage and if doesn't exist make it using collateralBrandKey.

Reply:
It could be included here without waiting for that PR but I won't request that.